fms/models/__init__.py

```diff
@@ -65,6 +66,47 @@ def list_variants(architecture: str):
     return list(__models[architecture].keys())
 
 
+def __maybe_infer_model_variant(
+    architecture: str,
+    variant: str,
+    model_path: Optional[str],
+    source: Optional[str],
+    **kwargs,
+) -> Tuple[str, str, Optional[str], Optional[str], Dict[str, Any]]:
+    """Infer the model variant configuration from different sources, currently only supported sources are hf"""
+    extra_kwargs = kwargs
+
+    if architecture in ("hf_pretrained", "hf_configured"):
+        logging.info(f"inferring model configuration from {variant}")
```

Comment:
should we make our logger a little bit more specific?

Reply:
as in add the module name at least: `logger = logging.getLogger(__name__)`

Reply:
fixed in latest commit